api/tsconfig.json

```diff
@@ -11,6 +11,6 @@
     "noUnusedParameters": false,
     "allowSyntheticDefaultImports": true
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "src/scripts/joiGenerator.ts"],
```

Comment:
isn't "src/**/*.ts" enough to include the "src/scripts/joiGenerator.ts" file too?

Reply:
you are right, this was added somehow by vscode. Already removed.